flocker/apiclient/test/test_client.py

```diff
@@ -751,11 +771,8 @@ def create_client(self):
         api_service.startService()
         self.addCleanup(api_service.stopService)
 
-        credential_set.copy_to(credentials_path, user=True)
-        return FlockerClient(reactor, b"127.0.0.1", self.port,
-                             credentials_path.child(b"cluster.crt"),
-                             credentials_path.child(b"user.crt"),
-                             credentials_path.child(b"user.key"))
+        credential_set.copy_to(self.credentials_path, user=True)
+        return self._create_flocker_client(False)
```

Comment:
:question: It'd be nice to use keyword argument here so that the reader doesn't have to go looking at the `create_flocker_client` signature.